pkgs/servers/polaris/default.nix

```
@@ -30,7 +31,15 @@ rustPlatform.buildRustPackage rec {
     '';
   };
 
-  cargoHash = "sha256-+4WN6TTIzVu3Jj0SfPq2jnYh0oWRo/C4qDMeJLrj1kk=";
+  cargoPatches = [
+    (fetchpatch { # https://github.com/agersant/polaris/pull/213
+      name = "bump-time-crate.patch";
+      url = "https://github.com/agersant/polaris/pull/213/commits/c1ef061c6913a9169e6daca7f49e51971298604e.patch";
```

Please use the upstream commit:

```suggestion
      url = "https://github.com/agersant/polaris/commit/f625c57d203bdd3f2d7fcd99ccce1032f04d9b91.patch";
```